application/models/requestsize.go

```diff
@@ -26,3 +28,14 @@ func (r RequestSize) isLargerOrSame(other RequestSize) bool {
 
 	return sizes[r] <= sizes[other]
 }
+
+func GetRequestSizeFromAPISize(apiSize api.RequestSize) RequestSize {
```

Comment:
Can `models` just use the type in `api`?

Reply:
I was thinking that since gql is relying on the models.RequestSize, it would be easier to wait until we have removed gql to make that change.

Reply:
yep, good call. I later ran into that myself